### ISSUE TITLE
Installation fails on macOS 10.15.6

### ISSUE THREAD

**Issue by HassanCorrigan** [andrewbranch/gatsby-remark-vscode#119]
Installation of this package seems to fail on macOS 10.15.6, using node v14.7.0 and npm v6.14.7. The oniguruma package is where the install seems to fail. Not sure if there is an issue with my config or if it's due to the package not being supported on macOS. 

```
 oniguruma@7.2.1 install /Users/User/Project
 node-gyp rebuild

No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/usr/local/Cellar/node/14.7.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/User/Project/node_modules/oniguruma
gyp ERR! node -v v14.7.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN @pmmmwh/react-refresh-webpack-plugin@0.3.3 requires a peer of react-refresh@^0.8.2 but none is installed. You must install peer dependencies yourself.
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN gatsby-interface@0.0.166 requires a peer of gatsby@2.6.0 but none is installed. You must install peer dependencies yourself.
npm WARN gatsby-interface@0.0.166 requires a peer of react@16.8.1 but none is installed. You must install peer dependencies yourself.
npm WARN gatsby-interface@0.0.166 requires a peer of react-dom@16.8.1 but none is installed. You must install peer dependencies yourself.
npm WARN theme-ui@0.2.52 requires a peer of @mdx-js/react@^1.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! oniguruma@7.2.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the oniguruma@7.2.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

The error log contents:
```
832 warn @pmmmwh/react-refresh-webpack-plugin@0.3.3 requires a peer of react-refresh@^0.8.2 but none is installed. You must install peer dependencies yourself.
833 warn acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
834 warn gatsby-interface@0.0.166 requires a peer of gatsby@2.6.0 but none is installed. You must install peer dependencies yourself.
835 warn gatsby-interface@0.0.166 requires a peer of react@16.8.1 but none is installed. You must install peer dependencies yourself.
836 warn gatsby-interface@0.0.166 requires a peer of react-dom@16.8.1 but none is installed. You must install peer dependencies yourself.
837 warn theme-ui@0.2.52 requires a peer of @mdx-js/react@^1.0.0 but none is installed. You must install peer dependencies yourself.
838 warn tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
839 verbose stack Error: oniguruma@7.2.1 install: `node-gyp rebuild`
839 verbose stack Exit status 1
839 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
839 verbose stack     at EventEmitter.emit (events.js:314:20)
839 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
839 verbose stack     at ChildProcess.emit (events.js:314:20)
839 verbose stack     at maybeClose (internal/child_process.js:1051:16)
839 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
840 verbose pkgid oniguruma@7.2.1
841 verbose cwd /Users/User/Project
842 verbose Darwin 19.6.0
843 verbose argv "/usr/local/Cellar/node/14.7.0/bin/node" "/usr/local/bin/npm" "install" "gatsby-remark-vscode"
844 verbose node v14.7.0
845 verbose npm  v6.14.7
846 error code ELIFECYCLE
847 error errno 1
848 error oniguruma@7.2.1 install: `node-gyp rebuild`
848 error Exit status 1
849 error Failed at the oniguruma@7.2.1 install script.
849 error This is probably not a problem with npm. There is likely additional logging output above.
850 verbose exit [ 1, true ]
```

**Comment by alvarezskinner** [comment]
Same issue on WSL2 (Ubuntu), doesn't seem to be something linked to just MacOS

**Comment by andrewbranch** [comment]
🤦 At some point I got bulk subscribed to a billion GitHub repos at work and I had to unwatch everything, and I forgot to go back and re-watch the things I actually maintain and care about, like this repo. Sorry for the delay.

A while back I decided it would be best if I could use the more portable onigasm (WASM port of oniguruma) by default, but it was blocked on https://github.com/microsoft/vscode-textmate/issues/79. I just noticed that’s fixed now, so I’m looking forward to removing oniguruma as a dependency altogether. 🎉 It causes issues every time there’s a new major version of node, too. I’ll take a look at that perhaps this weekend.
